src/sentry/migrations/0527_backfill_next_checkin_latest.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 3.2.20 on 2023-08-01 20:51
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_next_checkin_latest(apps, schema_editor):
+    MonitorEnvironment = apps.get_model("sentry", "MonitorEnvironment")
+
+    for monitor_environment in RangeQuerySetWrapperWithProgressBar(
+        MonitorEnvironment.objects.all()
+    ):
+        if monitor_environment.next_checkin_latest is not None:
+            continue
+
+        monitor_environment.next_checkin_latest = monitor_environment.next_checkin
```

Comment:
Looks like `next_checkin` can be null, not sure if it is in practice?

Reply:
it is for newly created monitor environments. will double check redash. but the main thing is pulling any old values into the new field if they haven't been updated. if both are null that's fine.

Reply:
yeah there are some where it's null

Reply:
updated logic